src/main/config.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <sstream>
+#include "gflags/gflags.h"
+#include "config.hpp"
+
+extern "C" {
+	#include "exec_monitor.h"
+}
+
+/*
+	FLAGS SECTION
+*/
+
+// EXEC FLAG - MONITOR PROCESS EXECUTION
+DEFINE_bool(exec, false, "Monitors process executions.");
+DEFINE_string(ppid, "", "Specifies the ppid(s) of the processes to monitor using comma-separated values.");
```

Comment:
Maybe your description could use "parent PID" instead of "ppid" to be a bit more informative